frontend/src/static/js/v_summary.js

```diff
@@ -724,7 +801,7 @@ window.vSummary = {
       filtered = this.sortFiltered(filtered, filterControl);
 
       if (zIsMerge) {
-        this.mergeGroup(filtered);
+        this.mergeGroupByIndex(filtered, 0);
```

Comment:
Is this still applicable?

Reply:
Sorry I missed this out. But this means that I still need `mergeGroupByIndex` because here merging has to be done regardless of the `mergedGroupsMap` status.